src/components/Header/Header.js

```diff
@@ -23,6 +25,7 @@ const Header = ({ isLoggedIn, setIsLoggedIn }) => {
 
   const clearLocalStorage = () => {
     localStorage.clear()
+    client.resetStore()
```

Comment:
Grateful the problem of the cache overwriting our current session was a fairly easy fix!

Reply:
yay